packages/kubernetes-1.23/kubernetes-1.23.spec

```diff
@@ -103,6 +103,32 @@ Conflicts: (%{_cross_os}image-feature(no-fips) or %{_cross_os}kubelet-1.23-bin)
 %description -n %{_cross_os}kubelet-1.23-fips-bin
 %{summary}.
 
+%package -n %{_cross_os}kube-proxy-1.23
+Summary: Container cluster node proxy
+Requires: %{_cross_os}kubelet-1.23
+Requires: %{_cross_os}kube-proxy-1.23(binaries)
+
+%description -n %{_cross_os}kube-proxy-1.23
+%{summary}.
+
+%package -n %{_cross_os}kube-proxy-1.23-bin
+Summary: Container cluster node proxy binaries
+Provides: %{_cross_os}kube-proxy-1.23(binaries)
+Requires: (%{_cross_os}image-feature(no-fips) and %{_cross_os}kube-proxy-1.23)
+Conflicts: (%{_cross_os}image-feature(fips) or %{_cross_os}kube-proxy-1.23-fips-bin)
+
+%description -n %{_cross_os}kube-proxy-1.23-bin
+%{summary}.
+
+%package -n %{_cross_os}kube-proxy-1.23-fips-bin
+Summary: Container cluster node proxy binaries, FIPS edition
+Provides: %{_cross_os}kube-proxy-1.23(binaries)
+Requires: (%{_cross_os}image-feature(fips) and %{_cross_os}kube-proxy-1.23)
```

Comment:
One question, how will downstreams use this? Do they have to start this as a systemd service or will this binary be found by the `kubelet` somehow and it will call it? If the former, will it make sense to provide the systemd service that starts `kube-proxy`?